server/src/main/java/org/apache/seata/server/controller/ClusterController.java

```diff
@@ -123,13 +122,11 @@ public MetadataResponse cluster(String group) {
     }
 
     @PostMapping("/watch")
-    public void watch(HttpServletRequest request, @RequestParam Map<String, Object> groupTerms,
-        @RequestParam(defaultValue = "28000") int timeout) {
-        AsyncContext context = request.startAsync();
-        context.setTimeout(0L);
+    public void watch(Channel channel, @RequestParam Map<String, Object> groupTerms,
```

Comment:
保持向下兼容,要保留HttpServletRequest request,因为当前版本spring-web还没去除,所以HttpServletRequest request还要留着,有些接口会从7091访问进来
To maintain backward compatibility, keep the HttpServletRequest request, because the current version of spring-web has not been removed, so the HttpServletRequest request will remain, and some interfaces will be accessed from 7091